config/seata-config-apollo/src/main/java/io/seata/config/apollo/ApolloConfiguration.java

```diff
@@ -176,6 +180,12 @@ private void readyApolloConfig() {
                 System.setProperty(PROP_APOLLO_SECRET, secretKey);
             }
         }
+        if (!properties.containsKey(APOLLO_CLUSTER)) {
+            System.setProperty(PROP_APOLLO_CLUSTER, FILE_CONFIG.getConfig(getApolloCluster()));
```

Comment:
If the value is ""?

Reply:
It is "default",the default value of apollo.

Reply:
It is "default",the default value of apollo.

Reply:
> It is "default",the default value of apollo.

I mean, if it's empty, don't set it

Reply:
The apollo client will set 'default',if it's empty.

Reply:
The apollo client will set 'default',if it's empty.